OSpackages.sh

```diff
@@ -226,15 +224,34 @@ function MainInstall {
    YUMCMD="yum --nogpgcheck --installroot=${CHROOTMNT} "
    YUMCMD+="--disablerepo=* --enablerepo=${OSREPOS} install -y "
 
-   # Stub...
-   echo "${RPMFILE}" > /dev/null 2>&1
-
-   # Expand the "core" RPM group and store as array
-   mapfile -t INCLUDEPKGS < <(
-      yum groupinfo "${RPMGRP}" 2>&1 | \
-      sed -n '/Mandatory/,/Optional Packages:/p' | \
-      sed -e '/^ [A-Z]/d' -e 's/^[[:space:]]*[-=+[:space:]]//'
-   )
+   # If RPM-file not specified, use a group from repo metadata
+   if [[ ${RPMFILE} == "UNDEF" ]]
+   then
+      # Expand the "core" RPM group and store as array
+      mapfile -t INCLUDEPKGS < <(
+         yum groupinfo "${RPMGRP}" 2>&1 | \
```

Comment:
Can we check for an empty groupinfo result and errexit, similar to:

https://github.com/plus3it/AMIgen7/blob/master/ChrootBuild.sh#L280?

Reply:
Yeah... Probably a bad idea to assume that the operator knows the repo is borked.